admin/src/app/Components/hotel-cards/hotel-cards.component.html

```diff
@@ -0,0 +1,30 @@
+<div class="cards">
+    <mat-card class="example-card" *ngFor="let hotel of hotels">
+        <mat-grid-list cols="5" rowHeight="60px">
+            <mat-grid-tile colspan=3 rowspan=1>
+                <mat-card-title>{{hotel.name}}</mat-card-title>
+            </mat-grid-tile>
+            <mat-grid-tile colspan=2 rowspan=3>
+                <img mat-card-xl-image src="https://material.angular.io/assets/img/examples/shiba2.jpg" >
+            </mat-grid-tile>
+            <mat-grid-tile colspan=3 rowspan=1>
+                <mat-card-subtitle>{{hotel.country}} - {{hotel.sity}}</mat-card-subtitle>
```

Comment:
Change in all other places please
```suggestion
                <mat-card-subtitle>{{ hotel.country }} - {{ hotel.sity }}</mat-card-subtitle>
```